Components/Home.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Alert, AppRegistry, Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { DrawerNavigator } from 'react-navigation';
+import { Dropdown } from 'react-native-material-dropdown';
+import { Header } from 'react-native-elements';
+import NavigationBar from 'react-native-navbar';
+
+export default class HomeScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      titleText: 'Welcome to BreakPath',
```

Comment:
Since neither of these will be changing at any point, nor are they fetched from the database, it's better to make this a dumb component that does not have any state, and move the text into the text fields